src/logging.rs

```diff
@@ -19,7 +19,7 @@ impl log::Log for ScreenLogger {
             "{} {} {}:{}] {}\n",
             time::get_time().sec,
             record.level(),
-            record.location().file().split("/").last().unwrap(),
+            record.location().file().split('/').last().unwrap(),
```

Comment:
what's the difference (to this code)?
or was this a suggestion by clippy?

Reply:
clippy